src/System.Net.Sockets/src/RefactoringShims/InteropShim.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Yes. It's included under ` <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">`, which triggers for Linux, OS X, and the various BSDs.